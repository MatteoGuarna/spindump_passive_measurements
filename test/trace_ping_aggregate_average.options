--textual --format json --aggregate-mode --average-mode --aggregate 10.30.0.167 195.140.0.0/16 --aggregate 10.30.0.167 212.16.100.0/24