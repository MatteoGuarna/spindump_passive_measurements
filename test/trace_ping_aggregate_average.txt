An IPv4 ICMP ping file to www.iki.fi. This test looks at the aggregate and averaging capabilities of Spindump.
