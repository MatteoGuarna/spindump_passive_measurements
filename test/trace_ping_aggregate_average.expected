[
{ "Event": "new", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "0 sessions", "State": "Static", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "new", "Type": "H2NET", "Addrs": ["10.30.0.167","212.16.100.0/24"], "Session": "0 sessions", "State": "Static", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283552230569, "State": "Static", "Right_rtt": 20850, "Avg_right_rtt": 20850, "Dev_right_rtt": 0, "Packets1": 1, "Packets2": 0, "Bytes1": 1428, "Bytes2": 0, "Bandwidth1": 1428, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283553233186, "State": "Static", "Right_rtt": 21935, "Avg_right_rtt": 21392, "Dev_right_rtt": 767, "Packets1": 2, "Packets2": 1, "Bytes1": 2856, "Bytes2": 1428, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283554237287, "State": "Static", "Right_rtt": 20800, "Avg_right_rtt": 21195, "Dev_right_rtt": 641, "Packets1": 3, "Packets2": 2, "Bytes1": 4284, "Bytes2": 2856, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283555240193, "State": "Static", "Right_rtt": 20877, "Avg_right_rtt": 21115, "Dev_right_rtt": 547, "Packets1": 4, "Packets2": 3, "Bytes1": 5712, "Bytes2": 4284, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283556240797, "State": "Static", "Right_rtt": 20691, "Avg_right_rtt": 21030, "Dev_right_rtt": 510, "Packets1": 5, "Packets2": 4, "Bytes1": 7140, "Bytes2": 5712, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283557247335, "State": "Static", "Right_rtt": 21979, "Avg_right_rtt": 21188, "Dev_right_rtt": 598, "Packets1": 6, "Packets2": 5, "Bytes1": 8568, "Bytes2": 7140, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283558246675, "State": "Static", "Right_rtt": 21136, "Avg_right_rtt": 21181, "Dev_right_rtt": 546, "Packets1": 7, "Packets2": 6, "Bytes1": 9996, "Bytes2": 8568, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283559247406, "State": "Static", "Right_rtt": 21303, "Avg_right_rtt": 21196, "Dev_right_rtt": 508, "Packets1": 8, "Packets2": 7, "Bytes1": 11424, "Bytes2": 9996, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283560252257, "State": "Static", "Right_rtt": 20900, "Avg_right_rtt": 21163, "Dev_right_rtt": 485, "Packets1": 9, "Packets2": 8, "Bytes1": 12852, "Bytes2": 11424, "Bandwidth1": 1428, "Bandwidth2": 2856 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283561256663, "State": "Static", "Right_rtt": 21875, "Avg_right_rtt": 21234, "Dev_right_rtt": 510, "Packets1": 10, "Packets2": 9, "Bytes1": 14280, "Bytes2": 12852, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283562257230, "State": "Static", "Right_rtt": 21737, "Avg_right_rtt": 21280, "Dev_right_rtt": 507, "Packets1": 11, "Packets2": 10, "Bytes1": 15708, "Bytes2": 14280, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283563263865, "State": "Static", "Right_rtt": 23762, "Avg_right_rtt": 21487, "Dev_right_rtt": 864, "Packets1": 12, "Packets2": 11, "Bytes1": 17136, "Bytes2": 15708, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283564261170, "State": "Static", "Right_rtt": 20664, "Avg_right_rtt": 21423, "Dev_right_rtt": 858, "Packets1": 13, "Packets2": 12, "Bytes1": 18564, "Bytes2": 17136, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283565264576, "State": "Static", "Right_rtt": 21179, "Avg_right_rtt": 21406, "Dev_right_rtt": 827, "Packets1": 14, "Packets2": 13, "Bytes1": 19992, "Bytes2": 18564, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283566269392, "State": "Static", "Right_rtt": 21638, "Avg_right_rtt": 21421, "Dev_right_rtt": 799, "Packets1": 15, "Packets2": 14, "Bytes1": 21420, "Bytes2": 19992, "Bandwidth1": 1428, "Bandwidth2": 2856 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283567272676, "State": "Static", "Right_rtt": 20882, "Avg_right_rtt": 21388, "Dev_right_rtt": 784, "Packets1": 16, "Packets2": 15, "Bytes1": 22848, "Bytes2": 21420, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283568277248, "State": "Static", "Right_rtt": 20854, "Avg_right_rtt": 21356, "Dev_right_rtt": 770, "Packets1": 17, "Packets2": 16, "Bytes1": 24276, "Bytes2": 22848, "Bandwidth1": 1428, "Bandwidth2": 1428 },
{ "Event": "measurement", "Type": "H2NET", "Addrs": ["10.30.0.167","195.140.0.0/16"], "Session": "1 sessions", "Ts": 1559283569284154, "State": "Static", "Right_rtt": 22510, "Avg_right_rtt": 21420, "Dev_right_rtt": 795, "Packets1": 18, "Packets2": 17, "Bytes1": 25704, "Bytes2": 24276, "Bandwidth1": 1428, "Bandwidth2": 1428 }
]
